tests/test_migrations.py

```diff
@@ -77,97 +70,3 @@ def model_label(self):
-            enterprise_customer_user=self.enterprise_customer_user
-        )
-        self.user_data_sharing_consent_audit = factories.UserDataSharingConsentAuditFactory(
-            user=self.enterprise_customer_user
-        )
-        self.migrate_to_origin()
-
-    def tearDown(self):
-        """
-        Make sure to migrate back to the origin.
-        """
-        super(MigrateToNewDataSharingConsentTest, self).setUp()
-        self.migrate_to_origin()
-
-    @ddt.data(
-        (True, UserDataSharingConsentAudit.NOT_SET),
-        (True, UserDataSharingConsentAudit.ENABLED),
-        (True, UserDataSharingConsentAudit.DISABLED),
-        (True, UserDataSharingConsentAudit.EXTERNALLY_MANAGED),
-        (False, UserDataSharingConsentAudit.NOT_SET),
-        (False, UserDataSharingConsentAudit.ENABLED),
-        (False, UserDataSharingConsentAudit.DISABLED),
-        (False, UserDataSharingConsentAudit.EXTERNALLY_MANAGED),
-        (None, UserDataSharingConsentAudit.NOT_SET),
-        (None, UserDataSharingConsentAudit.ENABLED),
-        (None, UserDataSharingConsentAudit.DISABLED),
-        (None, UserDataSharingConsentAudit.EXTERNALLY_MANAGED),
-    )
-    @ddt.unpack
-    @ignore_warning(DeprecationWarning)
-    def test_enrollment_consent_transfers(self, ece_consent_state, udsca_consent_state):
-        """
-        Test that ``EnterpriseCourseEnrollment``'s consent data transfers over to ``DataSharingConsent``.
-
-        We check ``UserDataSharingConsentAudit`` as a last resort for consent state.
-        """
-        self.enterprise_course_enrollment.consent_granted = ece_consent_state
-        self.enterprise_course_enrollment.save()
-        self.user_data_sharing_consent_audit.state = udsca_consent_state
-        self.user_data_sharing_consent_audit.save()
-
-        self.migrate_to_dest()
-
-        data_sharing_consent = DataSharingConsent.objects.all().first()
-        if self.enterprise_course_enrollment.consent_available:
-            self.assertTrue(data_sharing_consent.granted)
-        else:
-            self.assertFalse(data_sharing_consent.granted)
-
-    @ignore_warning(DeprecationWarning)
-    def test_duplicate_consent_doesnt_cause_error(self):
-        """
-        An existing ``DataSharingConsent`` row shouldn't cause a migration error -- it should update accordingly.
-        """
-        DataSharingConsent.objects.create(
-            username=self.enterprise_customer_user.username,
-            course_id=self.enterprise_course_enrollment.course_id,
-            enterprise_customer=self.enterprise_customer_user.enterprise_customer,
-            granted=False,
-        )
-
-        self.enterprise_course_enrollment.consent_granted = True
-        self.enterprise_course_enrollment.save()
-
-        self.migrate_to_dest()
-
-        data_sharing_consent = DataSharingConsent.objects.all().first()
-        assert data_sharing_consent.granted
```

Comment:
Although the migration this was testing obviously still exists, there should be no issue for new users of edx-enterprise, so there's no need to test that code anymore.